accounts/analyze-exams.js

```diff
@@ -1,9 +1,129 @@
-// After getting exam data with get-exams.js,
-// this goes through that info and gives some totals
+// After getting exam data with get-exams.js, this goes through the exams.json
+// file and writes totals to analyzed-exams.json. Be sure to update the two
+// numbers for linked accounts and certs claimed for accurate results. You will
+// need to get those from the database.
 
 const fs = require("fs");
 
-fs.readFile("exams.json", "utf8", (err, data) => {
+const examJsonFile = "exams.json";
+
+/*
+ * Get the two variables below in the Atlas UI. linkedAccounts can be found
+ * by checking the number of records in the MsUsername collection. certsClaimed
+ * can be found by doing an aggregation on the user collection by following
+ * these steps:
+ * 1. Go to the user collection
+ * 2. Click the aggregation tab
+ * 3. Turn off the preview on the top right
+ * 4. Add a new stage
+ * 5. Disable the stage with the toggle next to the Stage 1 dropdown
+ * 5. Put $match in the Stage 1 dropdown
+ * 6. Put this in the Stage 1 query body: { isFoundationalCShardCertV8: true }
+ * 7. Add another new stage
+ * 8. Disable the stage with the toggle next to the Stage 2 dropdown
+ * 9. Put $group in the Stage 2 dropdown
+ * 10. Put this in the Stage 2 query body: { _id: null, count: { $sum: 1 }}
+ * 11. Enable the preview at the top right
+ * 12. Enable Stage 1 and wait for it to finish, it only takes a few seconds
+ * 13. Enable Stage 2 to see the number of certifications claimed
+ */
+const linkedAccounts = 1;
+const certsClaimed = 1;
```

Comment:
```suggestion
const linkedAccounts = 0;
const certsClaimed = 0;
```

Since the validation below says 0 means you didn't add the values, shouldn't these be 0 initially to catch when they aren't updated?

Reply:
yes, whoops - thanks 🎉